src/stores/RootStore.tsx

```diff
@@ -51,6 +48,21 @@ const RootStore = types
     }
   }))
   .actions(self => ({
+    setActiveAssay(assayOrganelle: IOrganelleRef) {
+      self.activeAssay = assayOrganelle;
```

Comment:
Hmm, is the idea of an `activeAssay` now obsolete? I think you might be able to get rid of it entirely.